### DISCUSSION TITLE
Azure DevOps Pipeline - Variable Replacement Question

### DISCUSSION THREAD

**Discussion by crosscuttech** [IvanJosipovic/BlazorApplicationInsights#180] [General]
What task do you guys recommend for replacing the connectionString key in the JS file during the release pipeline?

**Comment by IvanJosipovic** [comment]
Hey,

For AZDO, I have used https://marketplace.visualstudio.com/items?itemName=qetza.replacetokens
This can also be done with PowerShell etc.

**Reply by crosscuttech** [reply]
Awesome, thanks for the quick response!
